printer.cfg

```diff
@@ -50,6 +50,10 @@
 # [include config/hardware/addons/filament_sensors/runout_sensor.cfg] #TODO: create file
 
 [include config/hardware/addons/accelerometers/adxl345.cfg]
+# [include config/hardware/addons/accelerometers/adxl345_rpi.cfg]
+# [include config/hardware/addons/accelerometers/adxl345_usb.cfg]
```

Comment:
What is the difference between this file and the `adxl345_rpi.cfg`and the `adxl345_usb.cfg` file ?

Reply:
Same answer.